crates/revive-env/src/lib.rs

```diff
@@ -42,16 +43,20 @@ impl ExtBuilder {
     pub fn build(self) -> sp_io::TestExternalities {
         sp_tracing::try_init_simple();
         let mut t = frame_system::GenesisConfig::<Runtime>::default().build_storage().unwrap();
+        let mut balance_genesis_config = self.balance_genesis_config;
+        balance_genesis_config.push((
```

Comment:
```
        pallet_revive::GenesisConfig::<Runtime>::default().assimilate_storage(&mut t).unwrap();
        ```
        
        I think that this part is missing, having this you do not need to set balance for `revive account_id`

Reply:
Done!